qutebrowser/browser/qutescheme.py

```diff
@@ -435,6 +435,20 @@ def qute_settings(url):
     return 'text/html', html
 
 
+@add_handler('bindings')
+def qute_bindings(url):
+    """Handler for qute://bindings View qute bindings."""
+    assert url
+
+    bindings = {}
+    for mode in "normal hint command insert passthrough".split():
```

Comment:
This shouldn't be hardcoded here - instead, you can probably use something like `set(config.val.bindings.default.keys()).union(config.val.bindings.commands)` (untested).